config/config.go

```diff
@@ -341,6 +343,7 @@ type CGRConfig struct {
 	statsCfg                 *StatSCfg                // Configuration for StatS
 	thresholdSCfg            *ThresholdSCfg           // configuration for ThresholdS
 	supplierSCfg             *SupplierSCfg            // configuration for SupplierS
+	LoaderProfiles           []*LoaderSConfig         // configuration for Loader
```

Comment:
please change to unexported value (loaderSProfiles) and add a method to export it in the end of the file.